src/Umbraco.Infrastructure/Manifest/ManifestParser.cs

```diff
@@ -53,13 +54,14 @@ public class ManifestParser : IManifestParser
             ILocalizationService localizationService,
             IJsonSerializer jsonSerializer,
             ILocalizedTextService localizedTextService,
-            IShortStringHelper shortStringHelper)
+            IShortStringHelper shortStringHelper,
```

Comment:
I don't think you need to inject `ILoggerFactory`, `IDataTypeService`, or `ILocalizationService`  in here now.